cmd/helm/install.go

```diff
@@ -239,7 +244,21 @@ func runInstall(args []string, client *action.Install, valueOpts *values.Options
 	}
 
 	client.Namespace = settings.Namespace()
-	return client.Run(chartRequested, vals)
+
+	// Create context and prepare the handle of SIGTERM
+	ctx := context.Background()
+	ctx, cancel := context.WithCancel(ctx)
+
+	// Handle SIGTERM
+	cSignal := make(chan os.Signal)
+	signal.Notify(cSignal, os.Interrupt, syscall.SIGTERM)
+	go func() {
+		<-cSignal
+		fmt.Fprintf(out, "Release %s has been cancel.\n", args[0])
```

Comment:
typo: cancelled*

Reply:
Resolved in commit https://github.com/helm/helm/pull/9180/commits/4bc901c95f3589b6d6b4b3bc0843347334cd0f68